packages/block-library/src/math/editor.scss

```
@@ -0,0 +1,5 @@
+@use "@wordpress/base-styles/variables" as *;
+
+.wp-block-math__textarea-control .components-textarea-control__input {
```

```suggestion
.wp-block-math__textarea-control textarea {
```

Nit: It might be best to avoid selectors that have the `.components-` prefix whenever possible.

---

Thanks for the suggestion! I've updated the selector to use `textarea` instead of `.components-textarea-control__input` to avoid depending on component-internal class names.